dataprofiler/profilers/profiler_options.py

```diff
@@ -932,6 +932,34 @@ def _validate_helper(self, variable_path="CorrelationOptions"):
         return errors
 
 
+class SyntheticDataOptions(BaseInspectorOptions):
```

Comment:
The paradigm we've tried to follow previously is for the Options to match the naming of the property or grouping of properties being calculate. In StructuredProfiler it is called `nan_replication_metrics`. I think it would make sense for the options to reflect as: `NanReplicationMetricsOptions`. I'm open to suggestion on this.

I do see a pathway for choosing all synth data options such that it turns off anything that doesn't need to be calculated for synth data.

Reply:
I'm open to changing as well. The idea putting it in `StructuredProfiler` is that it is specific to structured data for right now.  I'd rather just create a `SyntheticDataOptions` and not an options class for just this one `nan_replication_metrics`. 

I do see a pathway for `SyntheticDataOptions` as well.